### ISSUE TITLE
Google Hangouts leaves insert mode

### ISSUE THREAD

**Issue by rcorre** [qutebrowser/qutebrowser#3297]
1. `:open hangouts.google.com`
2. Click someone's name on the left
3. Press `i` to enter insert mode
4. Start typing
5. After a few keystrokes, I get kicked out of insert mode

hangouts.google.com has never worked _great_ with qutebrowser, but it has been functional enough until recently.
I can't remember exactly when this started, as I ignored it for a bit. I'm guessing some time in the last month?

```
qutebrowser v1.0.3
Git commit: 
Backend: QtWebEngine (Chromium 56.0.2924.122)

CPython: 3.6.3
Qt: 5.9.2
PyQt: 5.9.1

sip: 4.19.5
colorama: 0.3.9
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: 1.0.2 $Id$
attr: 17.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.21.0
QtNetwork SSL: OpenSSL 1.0.2l  25 May 2017

Style: QFusionStyle
Platform: Linux-4.9.61-1-lts-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt
```

**Comment by The-Compiler** [comment]
Can you try reproducing it in QupZilla? It smells similar to #3124, which is a QttWebEngine issue.

**Comment by rcorre** [comment]
It does repro with QupZilla 2.2.1

**Comment by The-Compiler** [comment]
I'm assuming it's a duplicate of #3124 then, which means it should go away with the next Qt release (hopefully [later this week](http://lists.qt-project.org/pipermail/releasing/2017-November/004586.html)).

**Comment by The-Compiler** [comment]
Duplicate of #3124

**Comment by rcorre** [comment]
I agree, probably a dup of #3124 

Relevant quote:

> This is probably [QTBUG-64082](https://bugreports.qt.io/browse/QTBUG-64082) which was introduced with Qt 5.9.2 and will be fixed with 5.9.3 and 5.10.
